CODEOWNERS

```diff
@@ -168,6 +168,8 @@ extensions/filters/http/oauth2 @rgs1 @derekargueta @snowp
 /*/extensions/matching/common_inputs/environment @snowp @donyu
 # user space socket pair and event
 /*/extensions/io_socket/user_space @lambdai @antoniovicente
+# io_uring-based IO socket
+/*/extensions/io_socket/io_uring @rojkov @antoniovicente
```

Comment:
Let's chat about over slack about CODEOWNERS.  I want to make sure I can satisfy this extension's ownership needs if my name is listed here :)

Reply:
Matt suggested the code should be in the core. Then CODEOWNERS will remain untouched :)